api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java

```diff
@@ -1,6 +1,5 @@
 package com.codeforcommunity.dto.map;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
Assuming this is an unused import?

Reply:
This was automatically removed with mvn spotless:apply

Reply:
Doesn't look like JsonProperty is used in that file so I think it's good